packages/tools/cookbook/src/utils/print-transactions-local.ts

```diff
@@ -0,0 +1,24 @@
+import type { ICoinCaps, ICommandBuilder, IPactCommand } from '@kadena/client';
+
+/**
+ * Blocking/sync call to submit a command for non-transactional execution.
+ * In a blockchain environment this would be a node-local “dirty read”
+ *
+ * @export
+ * @param {((ICommandBuilder<ICoinCaps> & IPactCommand)[])} signedTransactions
+ * @param {string} apiHost
+ * @return {Promise<void>}
+ */
+export async function printTransactionsLocal(
```

Comment:
In typescript jsdoc, named [tsdoc](https://tsdoc.org/) you don't need to add the type. Does eslint not give warnings about this?
Same goes for `@export`, because there is an `export` keyword

Reply:
Hmm I don't get any lint warnings, but I also wasn't getting any lint warnings for missing tsdoc :/

Reply:
Ah I see. It's only when it's being exported from the file that's in the `package.json`.`main` entry. On build, it checks the exports to see if there's a [release tag](https://api-extractor.com/pages/tsdoc/doc_comment_syntax/#release-tags). 

So for this it's not needed I think, as it's not exposed as public api.